vip_hci/var/shapes.py

```diff
@@ -372,7 +375,7 @@ def get_annulus_segments(array, inner_radius, width, nsegm=8, theta_init=0,
     rad = np.sqrt((xx - cx) ** 2 + (yy - cy) ** 2)
     phi = np.arctan2(yy - cy, xx - cx)
     phirot = phi % (twopi)
-    outer_radius = inner_radius + width
+    outer_radius = inner_radius + (width*optim_scale_fact)
```

Comment:
Let's start with this implementation of the optimization segments, it's definitively easier to code this way. We might try later on what I've proposed

Reply:
We could discuss this tomorrow. I'm back in the lab.